### DISCUSSION TITLE
v2.0.0-beta.2

### DISCUSSION THREAD

**Discussion by jameslittle230** [jameslittle230/stork#345] [Announcements]
This is the second beta release of the Stork 2.0.0 release branch. 

The beta signifier means that the release is documented and usable, though the API is not final and not all features have been implemented yet. Until 2.0.0 is released, the source code is stored in the [v2 branch](https://github.com/jameslittle230/stork/tree/v2) of the repository.

The documentation is available [here](https://63e05aa8ce4d5300089b2b8e--stork-search.netlify.app/).

---

This beta release includes the following changes compared to beta 1:

* Restores HTML parsing behavior with included and excluded selectors
* Updates the scoring algorithm for accuracy
* Includes search results for stemmed alternatives for specific words
* Reports build errors while documents are read, rather than batched at the end
* Fixes a bug where certain theme styles were not being properly set
* Fixes some bugs related to correctly reporting the loading state of the WASM blob and the search index within Javascript

---

The following artifacts have been uploaded to the Stork CDN:

https://files.stork-search.net/releases/v2.0.0-beta.2/stork.js
https://files.stork-search.net/releases/v2.0.0-beta.2/stork.wasm
https://files.stork-search.net/releases/v2.0.0-beta.2/stork.css
https://files.stork-search.net/releases/v2.0.0-beta.2/stork-ubuntu-20-04
https://files.stork-search.net/releases/v2.0.0-beta.2/stork-ubuntu-22-04
https://files.stork-search.net/releases/v2.0.0-beta.2/stork-macos-11
https://files.stork-search.net/releases/v2.0.0-beta.2/stork-macos-12
https://files.stork-search.net/releases/v2.0.0-beta.2/federalist.st
https://files.stork-search.net/releases/v2.0.0-beta.2/3b1b.st

<hr /><em>This discussion was created from the release <a href='https://github.com/jameslittle230/stork/releases/tag/v2.0.0-beta.2'>v2.0.0-beta.2</a>.</em>

**Comment by hrw** [comment]
Index is bigger than with v1.6.0 ;(

 8269803  search-index.st
 9569015  search-index.st-v2beta2
